lms/views/lti/basic_launch.py

```diff
@@ -148,36 +155,6 @@ def _show_document(self, document_url, assignment_extra=None):
             [self.course], self.request.lti_params
         )
 
-        # An assignment has been configured in the LMS as "gradable" if it has
-        # the `lis_outcome_service_url` param
-
-        assignment_gradable = self.request.product.plugin.misc.is_assignment_gradable(
-            self.request.lti_params
-        )
-
-        # Store lots of info
-        assignment = self._record_assignment(
-            document_url, extra=assignment_extra, is_gradable=assignment_gradable
-        )
-
```

Comment:
Inlined this.

A while ago we have a few things here but now it's just "_record_assignment_relationships" 

If we go back to "store lots" it's easy enough to add a private method.